### PR TITLE
Fixed Errant space in UEFI guest creation

### PR DESCRIPTION
**Pull request by murf0**
Created a windows 10 guest with tap interface
got error when ran `sh -x /usr/local/sbin/iohyve start win10`: 
`
bhyve -c 2 -H -w -m 8G -s 0,hostbridge -s 3,ahci-cd,/iohyve/ISO// -s '4,ahci-hd,/dev/zvol/zroot/iohyve/win10/disk0,sectorsize=512' -s 10, virtio-net,tap5 -s '11,fbuf,tcp=0.0.0.0:5900,w=800,h=600,wait' -s 12,xhci,tablet -u -s 31,lpc -l com1,/dev/nmdm1A -l bootrom,/iohyve/Firmware/BHYVE_UEFI.fd/BHYVE_UEFI.fd ioh-win10
user@host:~/iohyve # pci slot 10:0: unknown device ""
`
There should not be a space after `-s 10,`?
Removed the extra space and the guest started fine.

Sorry , But i did not investigate further how the code is different from the normal startup of non-uefi guests.

### CODE REVIEW

**Comment by murf0** [comment]
The global? $pci variable is empty before so the removed space is just created before. 
It also seems that multiple tap interfaces wont work either. (Adds spaces between)

`bhyve -c 2 -H -w -m 8G -s 0,hostbridge -s 3,ahci-cd,/iohyve/ISO// -s '4,ahci-hd,/dev/zvol/zroot/iohyve/win10/disk0,sectorsize=512' -s 10,virtio-net,tap5 virtio-net,tap6 -s '11,fbuf,tcp=0.0.0.0:5900,w=800,h=600,wait' -s 12,xhci,tablet -u -s 31,lpc -l com1,/dev/nmdm1A -l bootrom,/iohyve/Firmware/BHYVE_UEFI.fd/BHYVE_UEFI.fd ioh-win10`

**Comment by murf0** [comment]
Ok, so on uefi __guest_get_bhyve_cmd() is not called for uefi_guest, so uefi does not support multiple taps right now?
The PR only solves the issue for the single tap.

File: lib/ioh-guest [hunk 1]
```diff
@@ -576,7 +576,7 @@ __guest_uefi() {
 	# Create tap if needed
 	__tap_setup $dataset
 	# Add tap interface
-	local tappci="$pci $(__tap_pci $dataset)"
+	local tappci="$pci$(__tap_pci $dataset)"
```
Comment (CoolmanCZ) [inline lib/ioh-guest]: There is no definition of local pci variable, so $pci should be removed.
local tappci="$(__tap_pci $dataset)"

**Comment by CoolmanCZ** [comment]
Hi,

I don't have the experience with UEFI quest. I think, that multiple TAP interfaces should be:

-s 10:0,virtio-net,tap5
-s 10:1,virtio-net,tap6, mac=...

So we need to enhance the iohyve script.

From the man page of bhyve
-s	slot,emulation[,conf]
 slot	     pcislot[:function]	bus:pcislot:function
			     The pcislot value is 0 to 31.  The	optional
			     function value is 0 to 7.	The optional bus value
			     is	0 to 255.  If not specified, the function
			     value defaults to 0.  If not specified, the bus
			     value defaults to 0.

BR, Radek

**Comment by murf0** [comment]
Yes that is correct. This is just a quick-fix I PRed before reading up on the issue. 
I started working on using __guest_get_bhyve_cmd function to generate a pci line. I'll Close this PR and create a new when finished.

**Pull request closed without merging**